index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head><link rel="stylesheet" href="styles.css"></head>
+  <body>
+    
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
```

Comment:
It's a good practice to use a section semantic tag. Kindly use here &lt;section&gt; tag for the form

Reply:
The reason why you should use a section tag for the form is Below:

The section tag is used when requirements of two headers or footers or any other section of documents are needed. Section tag grouped the generic block of related contents. The main advantage of the section tag is, it is a semantic element, which describes its meaning to both the browser and the developer.